common/storage/sql/sqlite3/connection.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "common/storage/connection.h"
+#include "common/logger_helper.h"
+#include "common/storage/sql/defs.h"
+
+#include <sqlite3.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+#define CONNECTION_LOGGER_ID "sqlite3_connection_storage"
```

Comment:
could we name this `stor_sqlite3_conn` (so in descending order in the dependency tree, not ascending)